eclair-core/src/main/resources/reference.conf

```diff
@@ -23,6 +23,7 @@ eclair {
     rpcuser = "foo"
     rpcpassword = "bar"
     zmq = "tcp://127.0.0.1:29000"
+    max-unsync-blocks = 2147483647 // for low memory nodes like Raspbery Pi set this parameter to a small number, eg 2, 1, or even 0
```

Comment:
Why such a high value ? if your bitcoind has more headers than blocks then it's extremely likely that it's out of sync. I'd rather have a much smaller default (1 or 2)